src/pages/SignUp/SignUp.js

```diff
@@ -23,15 +23,25 @@ export default class Login extends Component {
   };
 
   checkAll = e => {
-    const { name, email, pw, address } = this.state;
+    const { name, email, password, address } = this.state;
 
     if (
       chkName(name) &&
       chkEmail(email) &&
-      chkPwd(pw) &&
+      chkPwd(password) &&
       chkAddress(address) === true
```

Comment:
```suggestion
      chkAddress(address)
```
이미 값들이 그 자체로 boolean일 것이라서 굳이 === true 작성해주지 않아도 될 것 같습니다